CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 4.2.0
+
+This is a major Teleport release with a focus on new features and bug fixes.
+
+### Improvements
+
+* Alpha: Enhanced Session Recording let's you know what's really happening during a Teleport Session. [Read the docs](https://gravitational.com/teleport/docs/ver/4.2/features/enhanced_session_recording/).
+* Alpha: Workflows API lets admins escalate RBAC roles. [Read the docs](https://gravitational.com/teleport/docs/ver/4.2/enterprise/#approval-workflows).
+* Beta: Teleport provides HA Support using Firestone and Google Cloud Storage using Google Cloud Platform. [Read the docs](https://gravitational.com/teleport/docs/ver/4.2/gcp_guide/).
+* Remote tctl execution is now possible. [Read the docs](https://gravitational.com/teleport/docs/ver/4.2/cli-docs/#tctl). [#1525](https://github.com/gravitational/teleport/issues/1525) [#2991](https://github.com/gravitational/teleport/issues/2991)
+
+### Fixes
+
+* Fixed issue in socks4 when rendering remote address [#3110](https://github.com/gravitational/teleport/issues/3110)
+
+### Documentation
+
+* Adopting root/leaf terminology for trusted clusters https://gravitational.com/teleport/docs/ver/4.2/trustedclusters/
+* Documented Teleport FedRAMP & FIPS Support https://gravitational.com/teleport/docs/ver/4.2/enterprise/ssh_fips/
```

Comment:
These links should be shortened for consistency with all the other links in the release notes.  We don't seem to use any raw URLs elsewhere.

Reply:
Addressed via a rebase & squash. 
3976997  has better links.

Reply:
I think anchoring with an absolute link to the specific docs version as we're doing here is fine.